packages/strapi-generate-api/templates/mongoose/controller.template

```diff
@@ -29,11 +29,13 @@ module.exports = {
    */
 
   findOne: async (ctx) => {
-    if (!ctx.params._id.match(/^[0-9a-fA-F]{24}$/)) {
+    const data = await strapi.services.blog.fetch(ctx.params);
```

Comment:
Also, what happens if I send a bad ID like `http://myapi.com/blog/john` where `john` is the ID? The following condition `if (!ctx.params._id.match(/^[0-9a-fA-F]{24}$/))` has been added because in my memory Mongo was returning an error.

Reply:
Cause `john` (nice example) would NEVER match the given regex ... So thats why i've changed that check